change/@fluentui-react-slider-3db1e642-9ac3-4cc6-b2c1-8d6f1cce0f12.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
```

Since this is changing actual code not just tests
```suggestion
  "type": "prerelease",
```